idaes/apps/caprese/examples/cstr_model.py

```diff
@@ -42,8 +43,10 @@
 
 # See if ipopt is available and set up solver
 if SolverFactory("ipopt").available():
-    solver = SolverFactory("ipopt")
-    solver.options = {"tol": 1e-6, "mu_init": 1e-8, "bound_push": 1e-8}
+    solver = get_solver(
+        solver="ipopt",
+        options={"tol": 1e-6, "mu_init": 1e-8, "bound_push": 1e-8},
```

Comment:
Just to be sure - this won't affect the options in other tests will it?

Reply:
No, the options just override whatever is in the config if anything.  They don't change the config.